vta/config/de10-nano_sample.json

```diff
@@ -0,0 +1,13 @@
+{
+  "TARGET" : "de10-nano",
```

Comment:
can we rename this to `de10nano` to keep targets un-hyphenated. same goes for the `json` file

Reply:
The TARGET name along with the file name has been change [here](https://github.com/dmlc/tvm/pull/3494/files#diff-9a4cecae2fe3f211a8fd6b3d59847d27)